warehouse/models/mart/gtfs_schedule_latest/_gtfs_schedule_latest.yml

```diff
@@ -11,14 +11,24 @@ models:
     meta:
       ckan.authority: https://gtfs.org/schedule/reference/#agencytxt
     columns:
-      - &primary_key
+      - &schedule_dim_pk
         name: key
         description: |
           Synthetic primary key constructed from `feed_key` and `_line_number`.
         tests:
-          - unique:
```

Comment:
Have we actually dealt with this? I thought we reintroduced this issue, I think without this filter the test will fail

Reply:
good question, it's fine now because line_number will always exist; the issue was null values being equal

Reply:
https://github.com/cal-itp/data-infra/pull/2504#issuecomment-1564919117